locales/en/messages.json

```diff
@@ -6075,7 +6075,7 @@
         "description": "One of the elements of the OSD"
     },
     "osdDescElementRemaningTimeEstimate" : {
```

Comment:
Same here :)

Reply:
i did not look: is .js correct or needs fix too?